stable/insights-agent/templates/nova/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.nova.enabled -}}
+{{- $_ := set . "Label" "nova" }}
```

Comment:
These 2 lines shouldn't be needed. If I remember right those are used by the CronJob templates.

Reply:
I think the top one should still be there. We don't want to create the configmap if nova isn't enabled.

But the second one I believe you are correct